app/views/layouts/_noscript_notice.html.erb

```diff
@@ -1,7 +1,7 @@
 <noscript>
   <div class="navbar" style="padding: 12px; margin-top: -20px; background: #ff2211; border-bottom: 1px solid #ffbbaa; text-align: center; text: black;">
     <div class="container">
-      Please enable Javascript for the site to function correctly. We promise not to do anything bad!
+      <%= t '.paragraph_1' %>
```

Comment:
Yeah! It definitely makes it easier to find the translations now that they
are broken up into files and structured that way. I owe it to the Rails
guide on i18n

On Tue, Apr 28, 2015, 11:02 PM Andrew Brown <notifications@github.com>
wrote:

> In app/views/layouts/_noscript_notice.html.erb
> <https://github.com/drusepth/Indent/pull/433#discussion_r29309044>:
>
> > @@ -1,7 +1,7 @@
> >  <noscript>
> >    <div class="navbar" style="padding: 12px; margin-top: -20px; background: #ff2211; border-bottom: 1px solid #ffbbaa; text-align: center; text: black;">
> >      <div class="container">
> > -      Please enable Javascript for the site to function correctly. We promise not to do anything bad!
> > +      <%= t '.paragraph_1' %>
>
> Oh wait, I see, it's per page in the translations file. That's pretty
> slick.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/drusepth/Indent/pull/433/files#r29309044>.
>